src/olympia/amo/management/commands/cron.py

```diff
@@ -0,0 +1,36 @@
+import sys
+from importlib import import_module
+
+from django.conf import settings
+from django.core.management.base import BaseCommand
+
+from olympia.core import logger
+
+
+log = logger.getLogger('z.cron')
+
+
+class Command(BaseCommand):
+    help = 'Run one of the predefined cron jobs'
+    args = '[name args...]'
+
+    def handle(self, *args, **opts):
+        if not args:
+            log.error("Cron called without args")
+            print 'These jobs are available:\n%s' % '\n'.join(
```

Comment:
`self.stderr.write()` is preferred (or just `log.error` alone if the logging configuration is correct)